pyconfig.h.in

```diff
@@ -54,6 +54,9 @@
 /* Define if getpgrp() must be called as getpgrp(0). */
 #undef GETPGRP_HAVE_ARG
 
+/* Define to 1 if you have the <> header file. */
+#undef HAVE_
+
```

Comment:
This generic declaration seems to be out of place.

Reply:
> Are you up for pinning down the offending line(s)?

No success for today. I tried to trace how `configure` and `Makefile.pre.in` treat these header-related undefs by searching for `HAVE_ALLOCA_H` and case-sensitive `ALLOCA`. No match found in these two files.

Looking through all these treatment spots we could see an entry with a chopped name bound to `HAVE_` and map it back to the corresponding `configure.ac` line.

Reply:
I hope that https://github.com/python/cpython/pull/105062#discussion_r1210698431 is correct.

Reply:
Resolved with 68bc05e5a63eabf166bec3e612a5ff624038b408